src/Symfony/Component/Validator/Constraints/File.php

```diff
@@ -91,7 +91,10 @@ public function __get($option)
         return parent::__get($option);
     }
 
-    public function __isset($option)
+    /**
+     * @return bool
```

Comment:
should be removed: the parent has it already (same below)

Reply:
With below you mean `Symfony\Component\VarDumper\Cloner\Data`? That class does not have a parent.